src/index.jsx

```diff
@@ -49,27 +66,81 @@ function Form() {
         setPref('app', app);
     }, [app])
 
+    const menuItems = Object.entries(AVAILABLE_APPS).map(([name, value]) => ({
+        text: value.title,
+        key: name
+    }));
+    
+    const handleMenuChange = (item) => {
+        if (item) {
+            setApp(item.key);
+        }
+    };
+
+    const changeOpenState = (item) => {
+        if(isDropdownOpen == true){
+            document.getElementById("root").style.height="190px";
```

Comment:
I'd prefer we use something like CSS classes here, rather than directly setting CSS properties in JS.

Reply:
If this becomes small after moving these to a css class toggle, this function can also be inline.